R/mc_stat.R

```diff
@@ -1,6 +1,88 @@
-
-mc_stat <- function(targets, flags="") {
+mc_stat <- function(targets, flags="", verbose = TRUE) {
   cmd <- paste("stat", flags, targets)
   cmd <- gsub("\\s+", " ", cmd)
-  mc(cmd)
+  mc(cmd, verbose = verbose)
+}
+
+mc_stat_metadata <- function(target) {
+  
+  targets <- paste0(collapse = " ", trimws(target)) #trimws(strsplit(target, " ")[[1]])
+  stat <- minioclient::mc(glue::glue("stat {targets} --json"), verbose = FALSE)$stdout 
+#  stat <- mc_stat(targets, flags = "--json", verbose = FALSE)$stdout 
```

Comment:
is this suggesting that maybe `mc_stat` should also use the glue syntax to better vectorize over `targets`?